contracts/payloads/IGP95/PayloadIGP95.sol

```diff
@@ -0,0 +1,888 @@
+                    supplyToken: address(0),
+                    borrowToken: USDC_ADDRESS,
+                    baseWithdrawalLimitInUSD: 0,
+                    baseBorrowLimitInUSD: 1_000_000, // $1M
+                    maxBorrowLimitInUSD: 2_000_000 // $2M
+                });
+
+                setVaultLimits(VAULT_PAXG_XAUT__USDC); // TYPE_2 => 122
+
+                VAULT_FACTORY.setVaultAuth(
+                    PAXG_XAUT__USDC_VAULT,
+                    TEAM_MULTISIG,
+                    false
+                );
+            }
+        }
+
+        {
+            // [TYPE 2] PAXG-XAUT<>USDT | smart collateral & normal debt
+            address PAXG_XAUT__USDT_VAULT = getVaultAddress(123);
+
+            {
+                VaultConfig memory VAULT_PAXG_XAUT__USDT = VaultConfig({
+                    vault: PAXG_XAUT__USDT_VAULT,
+                    vaultType: VAULT_TYPE.TYPE_2,
+                    supplyToken: address(0),
+                    borrowToken: USDT_ADDRESS,
+                    baseWithdrawalLimitInUSD: 0,
+                    baseBorrowLimitInUSD: 1_000_000, // $1M
+                    maxBorrowLimitInUSD: 2_000_000 // $2M
+                });
+
+                setVaultLimits(VAULT_PAXG_XAUT__USDT); // TYPE_2 => 123
+
+                VAULT_FACTORY.setVaultAuth(
+                    PAXG_XAUT__USDT_VAULT,
+                    TEAM_MULTISIG,
+                    false
+                );
+            }
+        }
+
+        {
+            // [TYPE 2] PAXG-XAUT<>GHO | smart collateral & normal debt
+            address PAXG_XAUT__GHO_VAULT = getVaultAddress(124);
+
+            {
+                VaultConfig memory VAULT_PAXG_XAUT__GHO = VaultConfig({
+                    vault: PAXG_XAUT__GHO_VAULT,
+                    vaultType: VAULT_TYPE.TYPE_2,
+                    supplyToken: address(0),
+                    borrowToken: GHO_ADDRESS,
+                    baseWithdrawalLimitInUSD: 0,
+                    baseBorrowLimitInUSD: 1_000_000, // $1M
+                    maxBorrowLimitInUSD: 2_000_000 // $2M
+                });
+
+                setVaultLimits(VAULT_PAXG_XAUT__GHO); // TYPE_2 => 124
+
+                VAULT_FACTORY.setVaultAuth(
+                    PAXG_XAUT__GHO_VAULT,
+                    TEAM_MULTISIG,
+                    false
+                );
+            }
+        }
+    }
+
+    // @notice Action 10: Update Borrow Limits for sUSDe/USDC
+    function action10() internal isActionSkippable(10) {
+        address sUSDe_USDC_VAULT = getVaultAddress(7);
+        {
+            VaultConfig memory VAULT_sUSDe_USDC = VaultConfig({
+                vault: sUSDe_USDC_VAULT,
+                vaultType: VAULT_TYPE.TYPE_1,
+                supplyToken: sUSDe_ADDRESS,
+                borrowToken: USDC_ADDRESS,
+                baseWithdrawalLimitInUSD: 0, // $0
+                baseBorrowLimitInUSD: 2_500, // $2.5k
+                maxBorrowLimitInUSD: 5_000 // $5k
```

Comment:
can keep this at 2.5k too

Reply:
also need to manually set to keep expand percent at 1% and expand duration 720hrs
![Uploading image.png…]()